src/zenml/cli/base.py

```diff
@@ -653,7 +653,14 @@ def info(
         cli_utils.print_user_info(user_info)
 
     if stack:
-        cli_utils.print_debug_stack()
+        try:
+            cli_utils.print_debug_stack()
+        except ModuleNotFoundError:
+            cli_utils.warning(
+                "Could not print debug stack information. Please make sure "
+                "you have the necessary dependencies and integrations "
+                "installed for all your stack components."
+            )
```

Comment:
Maybe partially print error info here, e.g. which module is missing to give some hint? Or, if possible, list the flavors set of this stack.

Reply:
Yeah ok that's a good idea.

Reply:
Added this now.